tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -34,6 +34,33 @@ __standard_vector_size__: int
 STANDARD: ExplainType
 ANALYZE: ExplainType
 
+sqlnull: DuckDBPyType
+boolean: DuckDBPyType
```

Comment:
If these are all constants, by Python convention the names should be ALL CAPS

Reply:
Oh I actually like that better, but I wanted to keep them in line with the `struct_type`, `list_type` methods.
Having one uppercase and the other lowercase feels less coherent :/